### DISCUSSION TITLE
Team Request - Form Feedback

### DISCUSSION THREAD

**Discussion by JJediny** [GSA-TTS/GitHub-Administration#21] [General]
Please discuss the current [DRAFT/WIP Team Request Form](https://github.com/GSA-TTS/GitHub-Administration/issues/new?assignees=%40GSA-TTS%2Ftts-admins&labels=team%2Crequest&projects=&template=2-Team.yml&title=team-name):
- Too Short?
- Too Long?
- What's Missing?
- ?

**Comment by akf** [comment]
Fields that ask for "comma-separated list of github users" probably should say whether those should be GitHub usernames or email addresses. 

Would something like the cg-contributors team go under *.tts.gsa.gov? If not, where? Or is that a special case that you wouldn't expect to handle through this form? 

Maybe instead of a checkbox under attestation, it should prompt for the System Owner's name, e.g.: 
>  _____, the System Owner, is aware of and approves of this team. 

I appreciate the link at the top of these forms to other types of requests.

**Comment by JJediny** [comment]
> cg-contributors team go under *.tts.gsa.gov?

We don't want to confuse the user management delegated to the https://github.com/cloud-gov org. Yes, the idea of `*.tts.gsa.gov` is to serve as a catch-all for non-system TTS projects (also not to be confused with [GSA's Parent Org's process](https://github.com/GSA/GitHub-Administration). I've added an email field as a catch-all for any delegation the system owner works out within their systems for user management. Thank you for the feedback @akf !
